### DISCUSSION TITLE
troubles when registering github student developer pack

### DISCUSSION THREAD

**Discussion by HuyHa689** [community/community#38400] [GitHub Education]
Excuse me. I wanna get student developer pack by my school email. However, there are several troubles as below. I tried so many ways to solve these problems but still failed. Anyone can help me, please?

Your application cannot be reviewed until you fix the following:
You are significantly more likely to be verified if you have completed your GitHub user profile with your full name and a short bio.
Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?

**Comment by samus-aran** [comment]
Hi @HuyHa689  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
